src/components/molecules/tabbed-navigation/tabbed-navigation.tsx

```diff
@@ -0,0 +1,28 @@
+import { ReactElement, useState } from 'react';
+import { TabProps } from './tab';
+
+import './tabbed-navigation.scss';
+
+type TabbedNavigationProps = {
+  initiallySelected?: number,
+  children: ReactElement<TabProps>[],
+};
+
+export const TabbedNavigation = ({ children, initiallySelected = 0 }: TabbedNavigationProps): JSX.Element => {
+  const [activeTab, setActiveTab] = useState(initiallySelected);
+
+  return (
+    <div className="tabbed-navigation">
+      <ul className="tabbed-navigation__tabs">
```

Comment:
Will need to aria labels etc to this for screen readers

Reply:
Never used them. Something we can pair on together?